File: 1-Iniciante/1036-Formula-de-Bhaskara/1036.c
#include <stdio.h>
# include <math.h>

int main() {
    
    double a, b, c;
    scanf ("%lf %lf %lf", &a, &b, &c);
    if (a == 0 || b * b - 4 * a * c < 0) {
        printf ("Impossivel calcular\n");
    }
    else {
        printf ("R1 = %.5lf\n", (- b + pow((b * b - 4 * a * c), 0.5)) / (2 * a));
        printf ("R2 = %.5lf\n", (- b - pow((b * b - 4 * a * c), 0.5)) / (2 * a));
    }
    
    return 0;
}

File: 1-Iniciante/1036-Formula-de-Bhaskara/1036.java
import java.util.Scanner;
import java.lang.Math;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        double a = scan.nextDouble();
        double b = scan.nextDouble();
        double c = scan.nextDouble();
        double dt = b*b-4*a*c;
        if (dt < 0 || a == 0) {
            System.out.println("Impossivel calcular");
        }
        else {
            System.out.printf("R1 = %.5f\n", (-b+Math.sqrt(dt))/(2*a));
            System.out.printf("R2 = %.5f\n", (-b-Math.sqrt(dt))/(2*a));
        }
    }
}